components/test/src/main/java/org/trellisldp/test/LdpBasicContainerTests.java

```diff
@@ -340,34 +340,42 @@ default void testDeleteContainer() {
         final RDF rdf = getInstance();
         final EntityTag etag;
 
+        // TODO remove the getChildLocation and setChildLocation methods from this interface
+        final String childResource;
+
+        try (final Response res = target(getContainerLocation()).request().post(entity("", TEXT_TURTLE))) {
+            assertAll("Check for an LDP-RS", checkRdfResponse(res, LDP.RDFSource, null));
+            childResource = res.getLocation().toString();
+        }
+
+        meanwhile();
+
         try (final Response res = target(getContainerLocation()).request().get()) {
             assertAll("Check for an LDP-BC", checkRdfResponse(res, LDP.BasicContainer, TEXT_TURTLE_TYPE));
             final IRI identifier = rdf.createIRI(getContainerLocation());
             final Graph g = readEntityAsGraph(res.getEntity(), getBaseURL(), TURTLE);
             assertAll("Verify the resulting graph", checkRdfGraph(g, identifier));
-            assertTrue(g.contains(identifier, LDP.contains, rdf.createIRI(getChildLocation())),
+            assertTrue(g.contains(identifier, LDP.contains, rdf.createIRI(childResource)),
                     "Check for the presence of an ldp:contains triple");
             etag = res.getEntityTag();
             assertTrue(etag.isWeak(), "Verify that the ETag is weak");
         }
 
-        meanwhile();
-
         // Delete one of the child resources
-        try (final Response res = target(getChildLocation()).request().delete()) {
+        try (final Response res = target(childResource).request().delete()) {
             assertEquals(SUCCESSFUL, res.getStatusInfo().getFamily(), "Check the response type");
         }
 
         // Try fetching the deleted resource
-        try (final Response res = target(getChildLocation()).request().get()) {
+        try (final Response res = target(childResource).request().get()) {
```

Comment:
I've finally gotten this test to run, but only by putting a `meanwhile();` above this line. You can doubtless see why-- the Cassandra backend is fully async and without the `meanwhile();` this section of test runs "too quickly" after the delete and still returns a 200.

We have a long-term problem with testing async backends, but `meanwhile()`, can we move forward with a `meanwhile();`? 😀

Reply:
Hm, got to take this back a bit-- the `meanwhile();` only works some of the time, not surprisingly.

Reply:
There was an underlying problem in my code, and we don't need any `meanwhile();` here.